mkdocs.yml

```diff
@@ -635,4 +633,4 @@ extra:
 
 plugins:
     - search
-    - macros
+    - macros
```

Comment:
nit: add new line at the end.

![image](https://github.com/aquasecurity/tracee/assets/3372117/94669f71-ef0a-4789-9068-e18aa6fa4cd6)